static/js/public/store-categories.js

```diff
@@ -72,4 +75,4 @@ function storeCategories() {
   }
 }
 
-export { storeCategories };
+export { storeCategories, getColour };
```

Comment:
It feels a bit confusing to have `getColour` exported from `store-categories.js`. 
It is used by categories, and apparently it's used elsewhere - maybe it would be good to have it as separate small module and import wherever needed?